### PR TITLE
Fix: txHash made consistent in logging

### LINKED ISSUE #846: tx hash logging needs to be more consistent throughout

### PR DESCRIPTION
**Pull request by shekhar2807**
# Description

- TxHash made consistent in logging,

Fixes #846

### CODE REVIEW

File: cmd/claimCommission.go [hunk 1]
```diff
@@ -56,8 +56,9 @@ func (*UtilsStruct) ClaimCommission(flagSet *pflag.FlagSet) {
 	if err != nil {
 		log.Fatal("Error in claiming stake reward: ", err)
 	}
+	txnHash := transactionUtils.Hash(txn)
 
-	err = razorUtils.WaitForBlockCompletion(client, transactionUtils.Hash(txn).String())
+	err = razorUtils.WaitForBlockCompletion(client, txnHash.String())
```
Comment (ashish10677) [inline cmd/claimCommission.go:61]: Use Hex() here as well.

File: cmd/dispute.go [hunk 2]
```diff
@@ -357,9 +359,9 @@ func GiveSorted(client *ethclient.Client, blockManager *bindings.BlockManager, t
 		}
 	}
 	log.Info("Calling GiveSorted...")
-	log.Info("Txn Hash: ", transactionUtils.Hash(txn))
+	log.Info("Txn Hash: ", txnHash.Hex())
 	giveSortedLeafIds = append(giveSortedLeafIds, int(leafId))
-	err = razorUtils.WaitForBlockCompletion(client, transactionUtils.Hash(txn).String())
+	err = razorUtils.WaitForBlockCompletion(client, txnHash.String())
```
Comment (ashish10677) [inline cmd/dispute.go:364]: Use Hex() here as well

File: cmd/modifyCollectionStatus.go [hunk 3]
```diff
@@ -104,11 +104,12 @@ func (*UtilsStruct) ModifyCollectionStatus(client *ethclient.Client, config type
 	txnOpts := razorUtils.GetTxnOpts(txnArgs)
 	log.Infof("Changing active status of collection: %d from %t to %t", modifyCollectionInput.CollectionId, !modifyCollectionInput.Status, modifyCollectionInput.Status)
 	txn, err := assetManagerUtils.SetCollectionStatus(client, txnOpts, modifyCollectionInput.Status, modifyCollectionInput.CollectionId)
+	txnHash := transactionUtils.Hash(txn)
 	if err != nil {
 		return core.NilHash, err
 	}
-	log.Info("Txn Hash: ", transactionUtils.Hash(txn).String())
-	return transactionUtils.Hash(txn), nil
+	log.Info("Txn Hash: ", txnHash.String())
```
Comment (ashish10677) [inline cmd/modifyCollectionStatus.go:111]: Use Hex() here

**Review by ashish10677 — CHANGES_REQUESTED** [review]
Don't use txHash.String() anywhere. Use txHash.Hex()

**Comment by shekhar2807** [comment]
> Don't use txHash.String() anywhere. Use txHash.Hex()

Done @ashish10677

**Review by ashish10677 — APPROVED** [review]

**Pull request merged**